package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "catbus",
+  "version": "2.0.0",
+  "description": "An off canvas side menu manager.",
+  "main": "public/javascript/catbus.coffee",
+  "dependencies": {
+    "harp": "^0.15.2"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "./node_modules/.bin/harp server"
```

Comment:
harp should be in your path when you run `npm start`. Also, I'd say harp is a devDep. No big deal.